rpcs3/Emu/RSX/CgBinaryFragmentProgram.cpp

```diff
@@ -231,7 +231,7 @@ void CgBinaryDisasm::TaskFP()
 {
 	m_size = 0;
 	u32* data = reinterpret_cast<u32*>(&m_buffer[m_offset]);
-	ensure(((m_buffer_size - m_offset) % sizeof(u32) == 0));
+	ensure((m_buffer_size - m_offset)  % sizeof(u32) == 0);
```

Comment:
```suggestion
	ensure((m_buffer_size - m_offset) % sizeof(u32) == 0);
```

Reply:
maybe this was the intention of the original author:
`ensure(((m_buffer_size - m_offset) % sizeof(u32)) == 0);`

Reply:
It all means the same, I personally prefer without parenthesses.